docker/ingestd/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG LSST_STACK_VERSION
+FROM lsstsqre/centos:${LSST_STACK_VERSION}
+
+WORKDIR /home/lsst
+
+ARG CTRL_INGESTD_GIT_VERSION
+RUN git clone https://github.com/lsst-dm/ctrl_ingestd && \ 
```

Comment:
Better to `eups distrib install` this if possible.  Then the whole `scons` command can go away.

Reply:
where are the instructions to get this to work via `eups distrib install`?

Reply:
ctrl_ingestd isn't part of distrib, so we can't do this.